recipes/opcache.rb

```diff
@@ -0,0 +1,33 @@
+#
+# Cookbook Name:: osl-php
+# Recipe:: opcache
+#
+# Copyright 2013, Oregon State University
```

Comment:
Where did you get this file from that has a copyright year of 2013?

Reply:
I copied the header from another recipe instead of using knife to create the recipe